util/deploy-comment.sh

```diff
@@ -5,8 +5,8 @@
 
 STAGING_URL="$1"
 
-REPO_DIR="$(dirname $0)"
-source "${REPO_DIR}/logging.sh"
+UTIL_DIR="$(dirname $0)"
```

Comment:
nit: here and elsewhere: prefer `$(dirname ${BASH_SOURCE[0]})` in case of varied invocation patterns.